opencog/atoms/bind/PatternLink.cc

```diff
@@ -45,13 +45,6 @@ void PatternLink::common_init(void)
 	                 _fixed, _virtual, _pat.black);
 	_num_virts = _virtual.size();
 
-	// unbundle_virtual does not handle connectives. Here, we assume that
-	// we are being run with the DefaultPatternMatchCB, and so we assume
-	// that the logical connectives are AndLink, OrLink and NotLink.
-	// Tweak the evaluatable_holders to reflect this.
-	std::set<Type> connectives({AND_LINK, OR_LINK, NOT_LINK});
-	trace_connectives(connectives, _pat.clauses);
```

Comment:
Removal of this was to allow this https://github.com/opencog/atomspace/issues/46#issuecomment-107873361 to work.

If types like SET_UNION_LINK, SET_INTERSECTION_LINK, and SET_COMPLEMENT_LINK should be used instead of AND_LINK, OR_LINK, etc, then I guess the "is_virtualable" function can be changed to look for them.